alluxio-ui/common/src/utilities/misc/bytesToString.tsx

```diff
@@ -0,0 +1,19 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+export const bytesToString = (bytes: number) => {
+  if (bytes === 0) {
+    return '0Byte';
+  }
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+  const i = parseInt('' + Math.floor(Math.log(bytes) / Math.log(1024)), 10);
```

Comment:
Why do we convert from float(?) to string and then back to int? Is there a way to just do the math and convert to int?